kasa/tests/conftest.py

```diff
@@ -573,7 +573,7 @@ async def mock_discover(self):
     yield discovery_data
 
 
-@pytest.fixture()
+@pytest.fixture(scope="session")
```

Comment:
Is this for performance or some other reason?

Reply:
Yeah, this could allow a single dummy initialization for the whole test session, but considering how miniscule the effect is on the whole runtime of our test suite, I'll remove it for now :-)

I think I had this idea of having a dummy device supporting ~everything we know to allow testing all interfaces against it, and in such case it might make sense to only construct a single instance for all the tests.